zulip_bots/zulip_bots/bots/github_detail/test_github_detail.py

```diff
@@ -89,3 +89,11 @@ def test_too_many_request(self) -> None:
 
         with self.mock_config_info(self.mock_config):
             self.verify_reply(request, bot_response)
+
+    def test_no_owner_repo(self) -> None:
+        mock_invalid_config = {'owner': '', 'repo': ''}
+        request = '/#1'
+        bot_response = 'Failed to detect owner and repository name.'
+
+        with self.mock_config_info(mock_invalid_config):
+            self.verify_reply(request, bot_response)
```

Comment:
Can you add two additional tests for the following cases:
* Owner and repo are specified in the config file, but not in the message.
* Owner and repo are specified in the message, but not in the config file.